[OSDOCS#17058]: DC-2: Image Mirroring and oc mirror Tooling-CQA-1 (#115599)
openshift/openshift-docs: modules/oc-mirror-migration-differences.adoc

[id="oc-mirror-migration-differences_{context}"]
= Changes from oc-mirror plugin v1 to v2

[role="_abstract"]

> Before migrating from oc-mirror plugin v1 to v2, you must review the updated mirroring workflow commands, which have changed to align with the new v2 workflow.

Reviewer: ```suggestion
Review the updated mirroring workflow commands before migrating from oc-mirror plugin v1 to v2 so that you understand what has changed.
```

Reviewer 2: not resolved
